dev/translator-vm.js

```diff
@@ -98,7 +99,8 @@ export class TranslatorVM {
             japaneseUtil: this._japaneseUtil,
             database: dictionaryDatabase
         });
-        const deinflectionReasons = JSON.parse(fs.readFileSync(path.join(dirname, '..', 'ext', 'data/deinflect.json'), {encoding: 'utf8'}));
+        const deinflectionReasons = parseJson(fs.readFileSync(path.join(dirname, '..', 'ext', 'data/deinflect.json'), {encoding: 'utf8'}));
+        // @ts-expect-error - This file will be removed
```

Comment:
I assume the PR for this will be coming real soon so np, but I guess it would be good to include the word TODO just in case things like this get forgotten, so that it's rediscoverable programatically

Reply:
I'll just mark this as draft again, I forgot that PR wasn't up or merged yet. That file will be completely removed.

Reply:
(turns out I'm dumb and forgot to delete that file)